deps/readline.lua

```diff
@@ -396,7 +396,7 @@ local keyHandlers =
     self:getHistory(10)
   end},
   -- Printable characters
-  {{function(key, char) return char > 31 and key:sub(1,1) or nil end}, function(self, consumedKeys)
+  {{function(key, char) return char > 31 and key or nil end}, function(self, consumedKeys)
```

Comment:
This should probably be `sub(key, 1, 1)` instead of removing the `sub` entirely. Removing the `sub` causes the entire line to get treated as printable whenever the first character is a printable character (e.g. 'a\027[D' should only consume 'a' and then interpret the rest as a left arrow key press, but without the sub, it will print the entire sequence instead).